docs/pages/tutorial/add-navigation.mdx

```diff
@@ -9,9 +9,16 @@ import { FileTree } from '~/ui/components/FileTree';
 import { ProgressTracker } from '~/ui/components/ProgressTracker';
 import { Step } from '~/ui/components/Step';
 import { CODE } from '~/ui/components/Text';
+import { VideoBoxLink } from '~/ui/components/VideoBoxLink';
 
 In this chapter, we'll learn Expo Router's fundamentals to create stack navigation and a bottom tab bar with two tabs.
 
+<VideoBoxLink
+  videoId="8336fcFV_T4"
```

Comment:
This video id is used twice, on different pages, with different metadata. Is it a mistake?

Reply:
Good catch! 😅